frontend/src/components/Action/ActionForm.tsx

```diff
@@ -0,0 +1,250 @@
+import React, { useEffect, useState } from 'react'
+
+import { PersonDetails } from '@equinor/fusion'
+import { DatePicker, SearchableDropdown, SearchableDropdownOption, Select } from '@equinor/fusion-components'
+import { Button, Icon, Switch, TextField, TextFieldProps, Typography } from '@equinor/eds-core-react'
+import { add, error_filled } from '@equinor/eds-icons'
+
+import { Action, Participant, Priority, Question } from '../../api/models'
+import { Grid } from '@material-ui/core'
+import { barrierToString } from '../../utils/EnumToString'
+import { DataToCreateAction } from '../../api/mutations'
+import styled from 'styled-components'
+import ActionNotesList from './ActionNotesList'
+
+const RightOrientedDate = styled(Typography)`
+    float: right;
+`
+
+const RightOrientedSwitch = styled(Switch)`
+    float: right;
+`
+
+type TextFieldChangeEvent = React.ChangeEvent<HTMLTextAreaElement> & React.ChangeEvent<HTMLInputElement>
+
+type Validity = Exclude<TextFieldProps['variant'], undefined | 'warning'>
+
+const ErrorIconComponent = <Icon size={16} data={error_filled} color="danger" />
```

Comment:
I know it might have been there already, but maybe: 

```suggestion
const ErrorIcon = <Icon size={16} data={error_filled} color="danger" />
```

Reply:
Yeah, it was there already but I can change it